website/docs/providers/proxy/forward_auth.mdx

```diff
@@ -72,6 +73,9 @@ server {
         # authentik-specific config
         auth_request        /akprox/auth;
         error_page          401 = @akprox_signin;
+        # For domain level, use the below error_page to redirect to your Authentik server with the full redirect path
+        # error_page          401 =302 https://id.xcii.cc/akprox/start?rd=$scheme://$http_host$request_uri;
```

Comment:
Please use the same `authentik.company` placeholder

Reply:
Doh! I even made mention to this.. just wanted to double check the config before posting... rookie mistake!